docs/src/main/sphinx/sql/refresh-materialized-view.rst

```diff
@@ -0,0 +1,28 @@
+=========================
+REFRESH MATERIALIZED VIEW
+=========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    REFRESH MATERIALIZED VIEW view_name
+
+Description
+-----------
+
+Refresh the data stored in the materialized view ``view_name``. Data is
+retrieved from the underlying tables accessed by the query that creates the
+materialized view.
+
+The specific implementation varies by connector used to create the materialized
+view. The simplest implementation drops the materialized view and creates it
+again. More complex implementation insert missing data and update changed data.
```

Comment:
I feel like talking about "complexity" adds an unnecessary dimension to this concept. Would rephrase as

"Some connectors drop the materialized view and create it again, while others insert missing data and update changed data."

Reply:
Currently the only implementation in Trino is Iceberg and that simply drops the existing table and creates a new one with the results of executing the MV query.

Reply:
The Trino Iceberg implementation does NOT drop the table. It deletes the data from the storage table and inserts the data that is the result of executing the materialized view query. 
Ref: https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java#L979

Reply:
Thanks for clarifying, i misunderstood the meaning of `executeDelete`

Reply:
I think we can leave this info out actually .. removing for now. We can add more details in the specific connector docs or here at a later stage.